go/adbc/drivermgr/arrow-adbc/adbc.h

```diff
@@ -152,15 +152,19 @@ struct ArrowArrayStream {
 // Storage class macros for Windows
 // Allow overriding/aliasing with application-defined macros
 #if !defined(ADBC_EXPORT)
-#if defined(_WIN32)
+#if defined_WIN32 || defined __CYGWIN__
```

Comment:
The updates to this macro are inspired by @paleolimbot work in https://github.com/apache/arrow-nanoarrow/pull/719